wkt-crs-v1/wktcrsv1.g4

```diff
@@ -0,0 +1,135 @@
+localcs:       LOCAL_CS LPAR name COMMA localdatum COMMA unit COMMA (axis COMMA)+ authority RPAR;
+datum:         DATUM LPAR name COMMA spheroid (COMMA towgs84)? (COMMA authority)? RPAR;
+vertdatum:     VERT_DATUM LPAR name COMMA type COMMA authority RPAR;
+localdatum:    LOCAL_DATUM LPAR name COMMA type (COMMA authority)? RPAR;
+spheroid:      SPHEROID LPAR name COMMA semiMajorAxis COMMA inverseFlattening (COMMA authority)? RPAR;
+towgs84:       TOWGS84 LPAR dXBF COMMA dYBF COMMA dZBF (COMMA rXBF COMMA rYBF COMMA rZBF (COMMA mBF)?)? RPAR;
+authority:     AUTHORITY LPAR authorityName COMMA code RPAR;
+primem:        PRIMEM LPAR name COMMA longitude (COMMA authority)? RPAR;
+unit:          UNIT LPAR name COMMA angularUnit (COMMA authority)? RPAR;
+axis:          AXIS LPAR name COMMA axisOrient RPAR;
+projection:    PROJECTION LPAR name (COMMA authority)? RPAR;
+parameter:     PARAMETER LPAR name COMMA value RPAR;
+authorityName: '"EPSG"' | '"ESRI"';
+axisOrient:
+    'EAST'
+    | 'WEST'
+    | 'NORTH'
+    | 'SOUTH'
+    | 'NORTH_EAST'
+    | 'NORTH_WEST'
+    | 'UP'
+    | 'DOWN'
+    | 'GEOCENTRIC_X'
+    | 'GEOCENTRIC_Y'
+    | 'GEOCENTRIC_Z'
+    | name;
+
+epsgCode:          PKEY | NUMBER;
+name:              TEXT;
+number:            NUMBER;
+type:              NUMBER;
+semiMajorAxis:     NUMBER;
+inverseFlattening: NUMBER;
+dXBF:              NUMBER;
+dYBF:              NUMBER;
+dZBF:              NUMBER;
+rXBF:              NUMBER;
+rYBF:              NUMBER;
+rZBF:              NUMBER;
+mBF:               NUMBER;
+code:              TEXT;
+longitude:         NUMBER;
+angularUnit:       NUMBER;
+value:             NUMBER;
+
+COMPD_CS:    C O M P D '_' C S;
+PROJCS:      P R O J C S;
+GEOGCS:      G E O G C S;
+DATUM:       D A T U M;
+SPHEROID:    S P H E R O I D;
+TOWGS84:     T O W G S '84';
+AUTHORITY:   A U T H O R I T Y;
+PRIMEM:      P R I M E M;
+UNIT:        U N I T;
+AXIS:        A X I S;
+PROJECTION:  P R O J E C T I O N;
+PARAMETER:   P A R A M E T E R;
+VERT_CS:     V E R T '_' C S;
+VERT_DATUM:  V E R T '_' D A T U M;
+GEOCCS:      G E O C C S;
+LOCAL_CS:    L O C A L '_' C S;
+LOCAL_DATUM: L O C A L '_' D A T U M;
+
+NUMBER:       PM? INT ('.' INT)? EXP?;
+TEXT:         '"' ('""' | ~'"')* '"';
+PKEY:         [A-Z] [0-9A-Za-z]+;
+COMMENT_LINE: '#' ~[\r\n]*;
+WS:           [ \r\n\t]+ -> skip;
+
+COMMA: ',';
+LPAR:  '[' | '(';
+RPAR:  ']' | ')';
+EQ:    '=';
+
+fragment INT: [0-9]+;
+fragment EXP: [Ee] PM? INT;
+fragment PM:  '+' | '-';
+
+fragment A: [aA];
+fragment B: [bB];
+fragment C: [cC];
+fragment D: [dD];
+fragment E: [eE];
+fragment F: [fF];
+fragment G: [gG];
+fragment H: [hH];
+fragment I: [iI];
+fragment J: [jJ];
+fragment K: [kK];
+fragment L: [lL];
+fragment M: [mM];
+fragment N: [nN];
+fragment O: [oO];
+fragment P: [pP];
+fragment Q: [qQ];
+fragment R: [rR];
+fragment S: [sS];
+fragment T: [tT];
+fragment U: [uU];
+fragment V: [vV];
+fragment W: [wW];
+fragment X: [xX];
+fragment Y: [yY];
+fragment Z: [zZ];
```

Comment:
Please use [caseInsensitive](https://github.com/antlr/antlr4/blob/master/doc/options.md#caseinsensitive) option instead of these fragments.

Reply:
ok, updated it.